.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Task
+about: Something needs to be done
+title: ""
+labels: ""
+assignees: ""
+---
+
+Please take the time to describe the problem in detail, so that other contributors can pick it up and implement or declare it done/obsolete in case if was completed out of band.
+
+**What has to be done**
+Describe what has to be done, ideally in declarative form - what does success look like.
+
+**How do you know it has to be done**
+Link any documents, Slack threads, quote personal communication, reference your personal domain expertise etc. explaining why you think this should be done.
+
+**Code pointers**
```

Comment:
Is this still relevant to this type of issue?

Reply:
Absolutely, please see explaining when use a feature vs. a task.